test/e2e/framework.go

```diff
@@ -643,22 +644,42 @@ func (data *TestData) deleteTestNamespace(timeout time.Duration) error {
 }
 
 // deployAntreaCommon deploys Antrea using kubectl on the control-plane Node.
-func (data *TestData) deployAntreaCommon(yamlFile string, extraOptions string, waitForAgentRollout bool) error {
+func (data *TestData) deployAntreaCommon(yamlFile string, extraOptions string, isAgent bool) error {
 	// TODO: use the K8s apiserver when server side apply is available?
 	// See https://kubernetes.io/docs/reference/using-api/api-concepts/#server-side-apply
-	rc, _, _, err := provider.RunCommandOnNode(controlPlaneNodeName(), fmt.Sprintf("kubectl apply %s -f %s", extraOptions, yamlFile))
-	if err != nil || rc != 0 {
-		return fmt.Errorf("error when deploying Antrea; is %s available on the control-plane Node?", yamlFile)
+	deployAgentYML := func(yml string) error {
+		rc, _, _, err := provider.RunCommandOnNode(controlPlaneNodeName(), fmt.Sprintf("kubectl apply %s -f %s", extraOptions, yml))
+		if err != nil || rc != 0 {
+			return fmt.Errorf("error when deploying Antrea; is %s available on the control-plane Node?", yml)
+		}
+		return nil
+	}
```

Comment:
this applies that regular Antrea manifest which is the controller + the agent + other k8s resources. Why is the function called `deployAgentYML`?